components/chart/index.tsx

```diff
@@ -20,6 +21,8 @@ function Chart({
   const rootContent = data[0];
   const [zoomLevel, setZoomLevel] = useState(100);
   const [selectedModules, setSelectedModules] = useState([]);
+  const [description, setDescription] = useState('');
+  const [name, setName] = useState('');
```

Comment:
I thought about this one again, I think it would be better if we set the module itself like
```suggestion
  const [selectedModule, setSelectedModule] = useState<NodeType>();
```

In that case, we can use any property from the node in the description component and we will avoid setting the state for every parameter we need to show in